myhpi/urls.py

```diff
@@ -25,6 +25,7 @@
     ),
     path("select2/", include("django_select2.urls")),
     path("__debug__/", include("debug_toolbar.urls")),
+    path(".well-known/security.txt", RedirectView.as_view(url="/static/security.txt")),
```

Comment:
as the static path is configurable as STATIC_URL in the .env/settings, you should also respect that here